modules/client/shared/src/main/scala/lucuma/itc/client/SpectroscopyQuery.scala

```diff
@@ -13,29 +13,26 @@ import io.circe.JsonObject
 
 object SpectroscopyQuery extends GraphQLOperation[Unit] {
 
-  type Data      = List[SpectroscopyResult]
+  type Data      = SpectroscopyResult
   type Variables = SpectroscopyModeInput
 
   override val document: String =
     """
       query Spectroscopy($spec: SpectroscopyModeInput!) {
-        spectroscopy(input: $spec) {
+        spectroscopyExposureTime(input: $spec) {
           serverVersion
           dataVersion
-          results {
-            itc {
-              ... on ItcSuccess {
-                exposures
-                exposureTime {
-                  microseconds
-                }
-                signalToNoise
-                resultType
-              }
-              ... on ItcError {
-                msg
-                resultType
+          result {
+            __typename
+            ... on ExposureEstimate {
+              exposures
+              exposureTime {
+                microseconds
               }
+              signalToNoise
+            }
+            ... on CalculationError {
+              msg
             }
```

Comment:
Don't we need to handle `SourceTooBright` as well?

Reply:
We do